extensions/sql-migration/src/wizard/databaseBackupPage.ts

```diff
@@ -465,8 +516,10 @@ export class DatabaseBackupPage extends MigrationWizardPage {
 					if ((<azdata.CategoryValue>this._blobContainerStorageAccountDropdown.value).displayName === constants.NO_STORAGE_ACCOUNT_FOUND) {
 						errors.push(constants.INVALID_STORAGE_ACCOUNT_ERROR);
 					}
-					if ((<azdata.CategoryValue>this._blobContainerBlobDropdown.value).displayName === constants.NO_BLOBCONTAINERS_FOUND) {
-						errors.push(constants.INVALID_BLOBCONTAINER_ERROR);
+					for (let i = 0; i < this._blobContainerDropdowns.length; i++) {
```

Comment:
Why do you need to iterate over multiple? Shouldn't this be the only one?

Reply:
I will break the loop after first invalid dropdown in the array.